.github/workflows/bump-version.yml

```diff
@@ -1,27 +1,27 @@
----
-name: Version Bump
+name: Bump version
 
 on:
   workflow_dispatch:
     inputs:
       version_number:
-        description: "New Version"
+        description: "New version number"
         required: true
 
 jobs:
   bump_version:
-    name: "Bump Version to ${{ github.event.inputs.version_number }}"
```

Comment:
Any reason we would no longer include the version number? Seems like it could be helpful when doing reviews on a failing workflow.

Reply:
Simply put this is a very long string, jammed in between other strings for this workflow. It actually makes the GitHub user experience worse so someone can't easily see in a summary view what portion is running. The version number is all over the place in this workflow's logs, and it's used for dispatch **only** -- meaning the user typed the version in. That user knows what the version is.